Makefile

```diff
@@ -40,7 +40,7 @@ test-full: lint
 
 .PHONY: test-integration
 test-integration: node_version
-	npx mocha --recursive test/integration --exit
+	env BCRYPT=no npx mocha --recursive test/integration --exit
```

Comment:
why is `env` required here?

Reply:
Possibly it isn't. I don't know of a reason why it would be. Mostly I was just trying to match the `test` rule:

https://github.com/getodk/central-backend/blob/f7a11f026612a57154eccddbad0cc0eeceb1df70/Makefile#L35-L36

I'd be happy to remove `env` here or in both places!